gas/testsuite/gas/riscv/vector-insns.s

```diff
@@ -2,6 +2,7 @@
 	vsetvli a0, a1, 0
 	vsetvli a0, a1, 0x7ff
 	vsetvli a0, a1, e16,m2,d4
+	vsetvli a0, a1, e1024, m8, d8
```

Comment:
Could you add more test case to cover all new values?

Reply:
OK, I have added the new cases for the new values (e256, e512, e1024).

As for the test cases, which can cover all possible usage, we will support them in another commit :)